main.py

```diff
@@ -1,7 +1,29 @@
-import init_django_orm  # noqa: F401
-
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Drama")
+    Actor.objects.create(first_name="George", last_name="Clooney")
+    Actor.objects.create(first_name="Keanu", last_name="Reeves")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
+    Actor.objects.create(first_name="Will", last_name="Smith")
+    Actor.objects.create(first_name="Jaden", last_name="Smith")
+    Actor.objects.create(first_name="Scarlett", last_name="Johansson")
+
+    Genre.objects.filter(name="Drama").update(name="Drama")
```

Comment:
The update operation for the genre 'Drama' is redundant. It should update 'Dramma' to 'Drama' instead of 'Drama' to 'Drama'.